test/main.js

```diff
@@ -1456,6 +1456,22 @@ describe('node-fetch', () => {
 		});
 	});
 
+	it('should reject if the request body stream emits an error', () => {
+		const url = `${base}inspect`;
+		const requestBody = new stream.PassThrough();
+		const options = {
+			method: 'POST',
+			body: requestBody
+		};
+		const expectedError = new Error('an error');
+		setImmediate(() => {
+			requestBody.emit('error', expectedError);
+		});
+		return fetch(url, options).catch(error => {
+			expect(error).to.equal(expectedError);
+		});
```

Comment:
if `fetch` doesn't reject, the test fails with the following error:
`Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves`

Reply:
Are you sure about that? If `fetch` resolves instead of rejects then the test will be considered completed? Since `done` isn't explicitly called within the `catch` function...

Reply:
you are right, fixed